src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx

```diff
@@ -27,7 +29,7 @@ export interface NewViabilityTestModalProps {
 
 export default function NewViabilityTestModal(props: NewViabilityTestModalProps): JSX.Element {
   const { onClose, open, accession, organization, user, reload } = props;
-  const newViabilityTest: ViabilityTestPostRequest = {};
+  const newViabilityTest: ViabilityTestPostRequest = { testResults: [] };
```

Comment:
default withdrawnByUserId to current user. Also will take care of contributor use-case.